### DISCUSSION TITLE
SOAK testing with artillery playwright, issue with MaxVusers

### DISCUSSION THREAD

**Discussion by JoshCollis** [artilleryio/artillery#1732] [Artillery]
Hi there,

I'm trying to run a soak test with artillery playwright, the goal is to have 30 concurrent users running the test for 30 minutes.  Each time a browser finishes a scenario, a new one should be spawned in it's place to keep the total active browsers at 30

I have done spike and ramp testing at 30 concurrent users, and I expected the following yml file to achieve what I'm looking for as per the documentation, but it does not keep 30 browsers open, it seems to only open 5 browsers at a time

Any idea why this would be happening, and how iI can achieve this goal?

Thanks,
Josh

[example.txt](https://github.com/artilleryio/artillery/files/10229074/example.txt)

**Comment by tchambers97** [comment]
Hi Josh! How long does your scenario take to run? I expect you need to bump up your `arrivalRate`. With users arriving one per second, the only way to keep 30 users going is if your scenario takes at least 30 seconds, if my math is correct.

**Comment by JoshCollis** [comment]
Hi there,

My scenario takes between 15 and 40 seconds to run depending on the system I'm testing

The issue is that the max users option is limiting it to 5 in this scenario, as in I get a lot of skipped users at the end

Once those 5 users finish it spawns new ones

On 14 Dec 2022 19:56, tchambers97 ***@***.***> wrote:

Hi Josh! How long does your scenario take to run? I expect you need to bump up your arrivalRate. With users arriving one per second, the only way to keep 30 users going is if your scenario takes at least 30 seconds, if my math is correct.

**Reply by juanig1** [reply]
Hi!
There was a similar report [recently](https://github.com/artilleryio/artillery/discussions/1711). 
There was a bug were `maxVusers` was lower than it should in some cases. It was fixed [here](https://github.com/artilleryio/artillery/pull/1724) but we still have to release it.

Thanks for the script! I'll try to reproduce it and let you know

**Reply by juanig1** [reply]
There are some workarounds in that previous discussion too, seems to be that same case

**Reply by JoshCollis** [reply]
Thanks for the info!
